src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TripleColon/TripleColonParser.cs

```diff
@@ -39,8 +39,8 @@ public override BlockState TryOpen(BlockProcessor processor)
 
             var extensionName = "triple-colon";
             Action<string> logError = (string message) => _context.LogError(
-                $"invalid-{extensionName}",
```

Comment:
+ @superyyrrzz @fenxu be aware of this change

Reply:
@herohua - I was actually wondering about changing this pattern... Is this okay for me to do?

Reply:
Docs build side should make some adjustment if you change this pattern, not for free.

Reply:
Okay, that makes sense. I will just undo this pattern for now then.